pandas/tests/extension/test_arrow.py

```diff
@@ -2875,3 +2875,15 @@ def test_duration_overflow_from_ndarray_containing_nat():
     result = ser_ts + ser_td
     expected = pd.Series([2, None], dtype=ArrowDtype(pa.timestamp("ns")))
     tm.assert_series_equal(result, expected)
+
+
+def test_infer_dtype_pyarrow_dtype(data):
+    res = lib.infer_dtype(data)
+    assert res != "unknown-array"
+
+    if data._hasna and res in ["floating", "datetime64", "timedelta64"]:
+        # FIXME: in infer_dtype pd.NA is not ignored in these cases even
+        #  with skipna=True in the list(data) check below
+        return
```

Comment:
Would we turn this into an xfail so we know it needs addressing?

Reply:
good idea, will update

Reply:
updated+green